core/commands/bootstrap.go

```diff
@@ -352,27 +344,56 @@ func bootstrapAdd(r repo.Repo, cfg *config.Config, peers []config.BootstrapPeer)
 	return addedList, nil
 }
 
-func bootstrapRemove(r repo.Repo, cfg *config.Config, toRemove []config.BootstrapPeer) ([]config.BootstrapPeer, error) {
-	removed := make([]config.BootstrapPeer, 0, len(toRemove))
-	keep := make([]config.BootstrapPeer, 0, len(cfg.Bootstrap))
+func bootstrapRemove(r repo.Repo, cfg *config.Config, toRemove []string) ([]string, error) {
```

Comment:
This now has a new feature: One can remove a peer `/ip4/.../tcp/.../p2p/QmFoo` with `ipfs bootstrap rm /p2p/QmFoo` (no transport).